e2e/sample.config.yaml

```diff
@@ -1,49 +1,18 @@
-# This file contains configuration for running e2e tests.
-# Many of these fields can be overridden with environment variables.
-# All fields that support this have the corresponding environment variable name in a comment beside the field.
+# This file contains a bare minimum configuration for running e2e tests.
 
----
-chains:
-  # the entry at index 0 corresponds to CHAIN_A
-- chainId: chainA-1
-  numValidators: 1
-  numFullNodes: 0
-  image: ghcr.io/cosmos/ibc-go-simd # override with CHAIN_IMAGE
-  tag: main # override with CHAIN_A_TAG
-  binary: simd # override with CHAIN_BINARY
-
-  # the entry at index 1 corresponds to CHAIN_B
-- chainId: chainB-1
-  numValidators: 1
-  numFullNodes: 0
-  image: ghcr.io/cosmos/ibc-go-simd # override with CHAIN_IMAGE
-  tag: main # override with CHAIN_B_TAG
-  binary: simd # override with CHAIN_BINARY
-
-activeRelayer: hermes # override with RELAYER_ID
-relayers:
-  - id: hermes
-    image: ghcr.io/informalsystems/hermes
-    tag: "1.10.0"
-  - id: rly
-    image: ghcr.io/cosmos/relayer
-    tag: "latest"
-  - id: hyperspace
-    image: ghcr.io/misko9/hyperspace
-    tag: "20231122v39"
+# | Environment Variable | Description                               | Default Value                |
+# |----------------------|-------------------------------------------|------------------------------|
+# | CHAIN_IMAGE          | The image that will be used for the chain | ghcr.io/cosmos/ibc-go-simd   |
+# | CHAIN_A_TAG          | The tag used for chain A                  | N/A                          |
+# | CHAIN_B_TAG          | The tag used for chain B                  | N/A                          |
+# | CHAIN_BINARY         | The binary used in the container          | simd                         |
+# | RELAYER_TAG          | The tag used for the relayer              | 1.10.0                       |
+# | RELAYER_ID           | The type of relayer to use (rly/hermes)   | hermes                       |
 
-cometbft:
-  logLevel: info
+# see sample.config.extended.yaml for a fully configured sample.
+# set env E2E_CONFIG_PATH to point to this file to use it.
 
-debug:
-  # setting this value to true will force log collection even if the test passes.
-  dumpLogs: false
-  # settings this value to true will keep the containers running after the test finishes.
-  keepContainers: true
-
-# Required only for upgrade tests.
-# Chain A will be upgraded the specified tag.
-# The plan name must be registered as an upgrade handler in the given tag.
-upgrade:
-  planName: ""
-  tag: ""
+---
+chains:
+  - tag: main # override with CHAIN_A_TAG
+  - tag: main # override with CHAIN_B_TAG
```

Comment:
technically this is all you actually need now to have a valid config file assuming you want to use all the defaults (hermes 1.10.0, simd etc. )